docs/slurm-dws-flex.md

```diff
@@ -1,5 +1,10 @@
 # Obtaining SlurmGCP nodes with DWS Flex
 
+> [!NOTE]
+> DWS Flex Start is currently in early development and undergoing extensive testing. While it
```

Comment:
nit: is it possible to add a warning ?

Reply:
I don't think it merits to be a warning. Machines would still be provisioned.